jwst/lib/set_telescope_pointing.py

```diff
@@ -992,4 +1104,45 @@ def _add_axis_3(model):
     wcsaxes = model.meta.wcsinfo.wcsaxes
     if wcsaxes is not None and wcsaxes == 3:
         model.meta.wcsinfo.ctype3 = "WAVE"
-        model.meta.wcsinfo.cunit3 = "um"
\ No newline at end of file
+        model.meta.wcsinfo.cunit3 = "um"
+
+
+def calc_rotation_matrix(angle, vparity=1):
+    """ Calculate the rotation matrix
+
+    Parameters
+    ----------
+    angle: float in radians
+        The angle to create the matrix
+
+    vparity: {1, -1}
+        The x-axis parity, usually taken from
+        the JWST SIAF parameter VIdlParity
+
+    Returns
+    -------
+    matrix: [cd1_1, cd1_2, cd2_1, cd2_2]
```

Comment:
While it doesn't affect the outcome of the function, we really shouldn't be referring to these values as "cd". In the FITS standard WCS the "CD" matrix elements contain both rotation and scale. What we're computing is just rotation, which is designated as the "PC" matrix.